pong-game.py

```diff
@@ -0,0 +1,142 @@
+# Pong game
+
+import turtle
+import os
+
+
+win = turtle.Screen()
+win.title("Pong Game - Jaz")
+win.bgcolor("black")
+win.setup(width=800, height=600)
+win.tracer(0)
+sound = os.system("")
+
+# Scores
+score_1 = 0
+score_2 = 0
+
+# Paddle 1
+paddle_1 = turtle.Turtle()
+
+paddle_1.speed(0)  # sets animation speed to max
+paddle_1.shape("square")
+paddle_1.color("white")
+paddle_1.shapesize(stretch_wid=5, stretch_len=1)
+paddle_1.penup()
+paddle_1.goto(-350, 0)
+
+
+# Paddle 2
+paddle_2 = turtle.Turtle()
+
+paddle_2.speed(0)  # sets animation speed to max
+paddle_2.shape("square")
+paddle_2.color("white")
+paddle_2.shapesize(stretch_wid=5, stretch_len=1)
+paddle_2.penup()
+paddle_2.goto(350, 0)
+
+
+# Ball
+ball = turtle.Turtle()
+
+ball.speed(0) # issue - ball speed is set to 0 lowest speed for turtle but still goes very fast
+ball.shape("circle")
+ball.color("white")
+ball.penup()
+ball.goto(0, 0)
+ball.dx = 2

```

Comment:
I think these attributes (`dx` and `dy`) actually overrride what speed() does. If you try to set these to a different number, this will change the speed of the ball

Reply:
yes this managed to sort the ball speed out i guess this is the speed as it is the move of the ball of the x and y axis

Reply:
lowest speed value i found is 0.5 for x and -0.5 for y